code/game/machinery/fax_machine.dm

```diff
@@ -255,90 +266,82 @@ var/list/alldepartments = list()
-	msg_ghost += "Receiving '[sentname]' via secure connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>"
-
-	GLOB.ProvostFaxes.Add("<a href='?FaxView=\ref[faxcontents]'>\[view message at [world.timeofday]\]</a> <a href='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];USCMFaxReply=\ref[Sender];originfax=\ref[originfax]'>REPLY</a>")
-	announce_fax(msg_admin, msg_ghost)
+	if(fax_paper_copy)
+		return
 
-/proc/cmb_fax(originfax, sent, sentname, mob/Sender)
-	var/datum/fax/faxcontents = new(sent)
-	GLOB.fax_contents += faxcontents
+	if(istype(original_fax, /obj/item/paper))
+		fax_paper_copy = original_fax
+		return
 
-	var/msg_admin = SPAN_NOTICE("<b><font color='#1b748c'>COLONIAL MARSHAL BUREAU FAX: </font>[key_name(Sender, 1)] ")
-	msg_admin += "(<A HREF='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];ahelp=mark=\ref[Sender]'>Mark</A>) (<A HREF='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];adminplayeropts=\ref[Sender]'>PP</A>) "
-	msg_admin += "(<A HREF='?_src_=vars;Vars=\ref[Sender]'>VV</A>) (<A HREF='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];subtlemessage=\ref[Sender]'>SM</A>) "
-	msg_admin += "(<A HREF='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];adminplayerobservejump=\ref[Sender]'>JMP</A>) "
-	msg_admin += "(<a href='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];CMBFaxReply=\ref[Sender];originfax=\ref[originfax]'>RPLY</a>)</b>: "
-	msg_admin += "Receiving '[sentname]' via encrypted connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>"
+	fax_paper_copy = new()
+	fax_paper_copy.name = original_fax.name
 
-	var/msg_ghost = SPAN_NOTICE("<b><font color='#1b748c'>COLONIAL MARSHAL BUREAU FAX: </font></b>")
-	msg_ghost += "Receiving '[sentname]' via encrypted connection ... <a href='?FaxView=\ref[faxcontents]'>view message</a>"
+	if(istype(original_fax, /obj/item/photo))
+		var/obj/item/photo/faxed_photo = original_fax
+		photo_list += list("tmp_photo.png" = faxed_photo.img )
+		fax_paper_copy.info  += "<img src='tmp_photo.png' width='100%'/>"
+		return
 
-	GLOB.CMBFaxes.Add("<a href='?FaxView=\ref[faxcontents]'>\[view message at [world.timeofday]\]</a> <a href='?_src_=admin_holder;[HrefToken(forceGlobal = TRUE)];CMBFaxReply=\ref[Sender];originfax=\ref[originfax]'>REPLY</a>")
-	announce_fax(msg_admin, msg_ghost)
+	// type bundle
+	var/obj/item/paper_bundle/papers = original_fax
+	for(var/content in 1 to papers.amount)
+		fax_paper_copy.info += "<h1>Page #[content]</h1>"
+		fax_paper_copy.info += "<hr>"
+		if(istype(papers[content], /obj/item/paper))
+			var/obj/item/paper/faxed_paper = papers[content] 
+			fax_paper_copy.info += faxed_paper.info
+		else // type photo
+			var/obj/item/photo/faxed_photo = papers[content]
+			if(!isicon(faxed_photo.img))
+				return
+			photo_list += list("tmp_photo[content].png" = (faxed_photo.img))
+			fax_paper_copy.info  += "<img src='tmp_photo[content].png' width='100%'/>"
```

Comment:
```suggestion
			fax_paper_copy.info  += "<img src='tmp_photo[content].png' width='192'/>"
```
100% just looks way too stretched and requires too much height when there's a fax that's wide (it also shouldn't be stretching on a printed piece of paper). We'll double it.